docs-2.0/3.ngql-guide/3.data-types/9.type-conversion.md

```diff
@@ -0,0 +1,24 @@
+# Type Conversion
+
+Converting an expression of a given type to another type is known as type-conversion. In nGQL, type conversion is divided into explicit conversion.
+<!-- Converting an expression of a given type to another type is known as type-conversion. In nGQL, type conversion is divided into implicit conversion and explicit conversion. -->
+
+<!-- ## Implicit type conversion
+
+Implicit conversions are automatically performed when a value is copied to a compatible type.
+
+1. Following types can implicitly converted to `bool`:
+
+   - The conversions from/to bool consider `false` equivalent to `0` for empty string types, and `true` is equivalent to all other values.
+   - The conversions from/to bool consider `false` equivalent to `0` for int types, and `true` is equivalent to all other values.
+   - The conversions from/to bool consider `false` equivalent to `0.0` for float types, and `true` is equivalent to all other values.
+
+2. `int` can implicitly converted to `double`. -->
```

Comment:
int` can implicitly converted to `double`. -> int` can be implicitly converted to `double`. 
Lacks "be".